entity-service-impl/src/main/java/org/hypertrace/entity/data/service/EntityDataServiceImpl.java

```diff
@@ -694,6 +720,58 @@ private JSONDocument convertEntityRelationshipToDocument(
     }
   }
 
+  private void sentDeleteNotification(String tenantId, java.util.Collection<Entity> entities) {
```

Comment:
oh, so you've already done most of what I was talking about in the other comment - we swallow errors. Since the generator is local to this service and this is its only caller, we can push these down inside.

Reply:
Ideally, a rename too would help to convey to the caller that this won't fail. Like `trySendDeleteNotification`.

Reply:
Renamed. Just cleaner to have a separate method